science/2022-03-03-science-blog-5/index_de.md

```diff
@@ -0,0 +1,898 @@
+
+### 4.1 Aus der Anzahl der Warnenden
+
+Eine erste Möglichkeit der Schätzung der aktiven Nutzenden (im engeren Sinne) ergibt sich aus einem Vergleich der Anzahl der Warnenden (positiv getestete Personen, die ihre Schlüssel über die CWA teilen) mit der Anzahl der dem RKI übermittelten Neuinfektionen in Deutschland. Die folgende Abbildung zeigt den nahezu parallelen Verlauf dieser Kennzahlen.
+
+<figure>
+    <img src="./plot_au_estimate_1_warnings_infections_de.png" class="figure-img img-fluid" alt="Warnende angepasst an Neuinfektionen."/>
+    <figcaption aria-hidden="true">Abb. 7: Warnende angepasst an Neuinfektionen.</figcaption>
+</figure>
+
+Hierbei wurde ein konstantes Verhältnis des Anteils der Warnenden unter den Neuinfizierten angenommen. Tatsächlich hat sich dieses Verhältnis jedoch über die Zeit entwickelt. Der Unterschied am Ende ist durch den fehlenden Meldeverzug bei den Warnenden (und der Datenbereitstellung über das CWA-Ökosystem) gegenüber dem etablierten Meldesystem zu erklären. Des Weiteren sollten wir bei den Neuinfektionen die jüngeren Altersgruppen nicht berücksichtigen, da diese wegen der Altersbeschränkung (Nutzungszulassung ab 16 Jahren) nicht als Nutzende der CWA angenommen werden. Ebenso hängt der genannte Anteil — und damit die Schätzung — von der Teilungsrate ab, d.h. welcher Anteil derjenigen, die einen Test über die Corona-Warn-App registrieren, ihr Testergebnis erhalten und dieses dann auch mit anderen teilen. Diese Raten sind in der folgenden Abbildung dargestellt.
+
+<figure>
+    <img src="./plot_au_estimate_1_rates_de.png" class="figure-img img-fluid" alt="Teilungsrate und Verhältnis von Warnenden zu Neuinfektionen (unter Berücksichtigung der Altersgrenze)."/>
+    <figcaption aria-hidden="true">Abb. 8: Teilungsrate und Verhältnis von Warnenden zu Neuinfektionen (unter Berücksichtigung der Altersgrenze).</figcaption>
+</figure>
+
+Wenn wir nun annehmen, dass sich das Verhältnis von Warnenden zu Neuinfektionen genauso verhält, wie das Verhältnis der aktiven Nutzenden (im engeren Sinne) zur (relevanten) Bevölkerung unter Berücksichtigung der Teilungsrate (wobei wir für die Bevölkerung 83 Mio. bzw. für die potenziellen Downloadenden 60 Mio. (d.h. [Personen im Alter von über 15 Jahren, die ein Smartphone besitzen](https://de.statista.com/statistik/daten/studie/198959/umfrage/anzahl-der-smartphonenutzer-in-deutschland-seit-2010/)) ansetzen):
+
+<figure>
+    <img src="./fig9_de.png" class="figure-img img-fluid" width=100% alt="Annahme des gleichen Verhaltens von Verhältnis von Warnenden zu Neuinfektionen und Verhältnis der aktiven Nutzenden (im engeren Sinne) zur (relevanten)Bevölkerung unter Berücksichtigung der Teilungsrate"/>
```

Comment:
"(relevanten)Bevölkerung" => "(relevanten) Bevölkerung" (space missing)

Reply:
Fixed in the latest commit, thanks!